resources/articles/s2s-auth.md

```diff
@@ -0,0 +1,121 @@
+:title Service-to-service auth
+
+:frontpage
+:category api-integration
+:aside
+## Table of Contents
+
+<spid-toc></spid-toc>
+
+## Prerequisites
+
+In order to make use of this information, you need to know your:
+
+- client ID
+- client secret
+
+## See also
+
+- [Obtaining tokens](/oauth/token/)
+- [Token introspection](/token-introspection/)
+
+:body
+
+In addition to the user auth facilities provided by the platform, SPiD also provides a mechanism for
+when two services need to communicate directly without a user present. In these situations, a service
```

Comment:
"when two services need to communicate directly without a user present" -> "when two services want to communicate directly with each other" (The general case of protected resources doesn't even care about user existence, so user presence is not a factor here :)).

Reply:
Fixed